blogs/data-mover/data-mover-1.2.0-overview.md

```diff
@@ -0,0 +1,134 @@
+	<li>Makes sure that the data movement workflow has the appropriate storage credentials</li>
+	<li>Performs the copy of VolumeSnapshotContent, CSI VolumeSnapshot and PersistentVolumeClaims from application namespace to OADP Operator namespace</li>
+	<li>Triggers the data movement process and subsequently performs the cleanup of extraneous resources created.</li>
+</ul>
+
+<h3><a href="https://github.com/migtools/velero-plugin-for-vsm">VSM PLUGIN</a>:</h3>
+
+<p dir="auto">The upstream Velero plugin for the VolumeSnapshotMover(VSM) controller is to facilitate CSI volumesnapshot data movement from an OpenShift cluster to object storage, and vice versa.</p>
+
+<h3><a href="https://github.com/migtools/volume-snapshot-mover/tree/master/config/crd/bases">VOLUMESNAPSHOTMOVER CUSTOM RESOURCE DEFINITIONS (CRDS)</a>:</h3>
+
+<h1>VSB and VSR</h1>
+<p>
+VSB and VSR are Kubernetes custom resources (CRs) that can be used to create backups and restore PersistentVolumeClaims (PVCs).
+</p>
+<h2>VolumeSnapshotBackup (VSB)</h2>
+<p>
+A VSB represents a snapshot of a PVC. It can be used to create a VSR, which can be used to restore the PVC from the snapshot.
+</p>
+<table>
+<thead>
+<tr>
+<th>Field</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>ProtectedNamespace</td>
+<td>The namespace where the Velero deployment is located, and PVC is copied to.</td>
+</tr>
+<tr>
+<td>ResticSecretRef</td>
+<td>A reference to a Secret CR that contains the credentials for accessing the restic repository, as well as a reference for the given BSL.</td>
+</tr>
+<tr>
+<td>VolumeSnapshotContent</td>
+<td>The name of the VolumeSnapshotContent CR that represents the snapshot.</td>
+</tr>
+</tbody>
+</table>
+<h2>VolumeSnapshotRestore (VSR)</h2>
+<p>
+A VSR represents a restore of a PVC from a snapshot. It can be used to restore a PVC from a VSB.
+</p>
+<table>
+<thead>
+<tr>
+<th>Field</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>ProtectedNamespace</td>
+<td>The namespace where the Velero deployment is located, and PVC is copied from.</td>
+</tr>
+<tr>
+<td>ResticSecretRef</td>
+<td>A reference to a Secret CR that contains the credentials for accessing the restic repository, as well as a reference for the given BSL.</td>
+</tr>
+<tr>
+<td>VolumeSnapshotMoverBackupRef</td>
+<td>A reference to a VolumeSnapshotBackup CR that represents the snapshot that will be used to restore the PVC.</td>
+</tr>
+</tbody>
+</table>
+
+
+<h2>Backup Process</h2>
+<div>
+	
+The CSI plugin is extended to facilitate the data movement of CSI VolumeSnapshots(VS) from the cluster to object storage. When Velero backup is triggered, a snapshot of the application volume is created, followed by the associated VolumeSnapshotContent(VSC). This leads to the creation of a VolumeSnapshotBackup(VSB), which triggers the Data Mover process as the VolumeSnapshotMover(VSM) controller begins reconciliation on these VSB instances.<br><br>
```

Comment:
I would add we create a VSB per VSC (1:1), just to add more color

Reply:
This leads to the creation of a VolumeSnapshotBackup(VSB) per VSC